assets/js/guides.js

```diff
@@ -0,0 +1,6 @@
+$(document).ready(function (){
+    $("h2").each(function(){
+        $(this).nextUntil("h2").andSelf().wrapAll('<div class="card guide-card"></div>')
```

Comment:
Ah, interesting workaround!

I _think_ if you look at the HTML output from AsciiDoc, you'll find that sections are already wrapped in an HTML element with a classname (`class="sect1"`, `class="sect2"`, etc), so you can probably do it with pure CSS in that case. For Markdown, I guess this is an OK fall back for the time being, though it may lead to the whole page "flashing" and re-rendering. We can workaround this in the future by having the docs sorcer pre-process the HTML or perhaps changing the design.